cmd/kubelet/app/server.go

```diff
@@ -1006,20 +1005,13 @@ func InitializeTLS(kf *options.KubeletFlags, kc *kubeletconfiginternal.KubeletCo
 		}
 	}
 
-	tlsCipherSuites, err := cliflag.TLSCipherSuites(kc.TLSCipherSuites)
+	tlsCipherSuites, insecureCipherNames, err := cliflag.TLSCipherSuites(kc.TLSCipherSuites)
 	if err != nil {
 		return nil, err
 	}
 
-	if len(tlsCipherSuites) > 0 {
-		insecureCiphers := flag.InsecureTLSCiphers()
```

Comment:
is the method `InsecureTLSCiphers` ever used after this PR? Should it be removed?